Ruby-Project/teacher_class.rb

```diff
@@ -1,7 +1,7 @@
 require './person_class'
-class Teacher < person
-  def initialize(specialization, _can_use_services)
-    super(specialization)
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- **[OPTIONAL]** The paramater `parent_permission` uses colon sign for assignment, so the way to assign it on the `super` argument is a little different from what we are used to.

  ```
  super(age, name, parent_permission: parent_permission)
  ```